auth/src/main/java/com/firebase/ui/auth/ui/phone/VerifyPhoneNumberFragment.java

```diff
@@ -159,24 +167,29 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                             PhoneNumberUtils.getPhoneNumber(formattedPhone);
                     setPhoneNumber(phoneNumberObj);
                     setCountryCode(phoneNumberObj);
+                    onNext();
```

Comment:
@samtstern Yup, sorry for not making that clearer. In the email sign-in flow, we move on to the next screen once the user selects an email from the picker. This line inside `onActivityResult` does the same thing as in the email sign-in flow where if a user selects an item from the picker we also "click" the Send Code button for them.